src/databricks/labs/remorph/reconcile/connectors/sql_server.py

```diff
@@ -0,0 +1,133 @@
+import re
+import logging
+from datetime import datetime
+
+from pyspark.errors import PySparkException
+from pyspark.sql import DataFrame, DataFrameReader, SparkSession
+from sqlglot import Dialect
+from pyspark.sql.functions import col
+
+from databricks.labs.remorph.reconcile.connectors.data_source import DataSource
+from databricks.labs.remorph.reconcile.connectors.jdbc_reader import JDBCReaderMixin
+from databricks.labs.remorph.reconcile.connectors.secrets import SecretsMixin
+from databricks.labs.remorph.reconcile.recon_config import JdbcReaderOptions, Schema
+from databricks.sdk import WorkspaceClient
+
+logger = logging.getLogger(__name__)
+
+
+class SqlServerDataSource(DataSource, SecretsMixin, JDBCReaderMixin):
+    _DRIVER = "sqlserver"
+    _SCHEMA_QUERY = """SELECT 
```

Comment:
move it as top-level file constant to reduce nesting

Reply:
Noted

Reply:
not resolved yet